test/usecases/implicit_argument/simulate.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from neuron import gui, h
+from neuron.units import ms
+
+
+def test_cacur():
+    nseg = 1
+
+    s = h.Section()
+    s.insert("cacur")
+    s.nseg = nseg
+
+    v_hoc = h.Vector().record(s(0.5)._ref_v)
+    t_hoc = h.Vector().record(h._ref_t)
+
+    h.stdinit()
+    h.tstop = 100.0 * ms
+    h.run()
+
+    v = np.array(v_hoc.as_numpy())
+    t = np.array(t_hoc.as_numpy())
+
+    return t, v
+
+
+def check_solution(t, v):
+    # solution is an affine function until t = 1 (ms), afterwards is a constant
+    # (1000 - 65)
+    solution = 1000 * t - 65
```

Comment:
You can make it terser if you want by using `np.min(t, 1)`. The comment merely states literally what's written in Python, it can go (so it wont become outdated).

Reply:
`np.minimum`, but yes I agree.